clients/banking/src/components/TransactionDetail.tsx

```diff
@@ -148,6 +148,27 @@ export const TransactionDetail = ({ transaction, large }: Props) => {
     />
   );
 
+  const referenceToDisplay = (ref: string) => (
```

Comment:
nit: could you use an other name than ref ? it's a bit confusing to me

Reply:
maybe also rename to `renderReferenceToDisplay` as it's a function